libs/ledger-live-common/src/__tests__/test-helpers/bridgeSetupUseMocks.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
+import nock from "nock";
+import { URL } from "url";
+import { MockContent } from "./types";
+import { promises as fsp } from "fs";
+
+/* Loop through each mock and apply them to nock */
+export async function initBackendMocks() {
+  const root = `${__dirname}/bridgeMocks`;
```

Comment:
Should we categorise files into different sub folders by family or coin instead of putting all mock files in a single folder?

Reply:
I tried but since we are "sniffing" globally we either have to :
- Find the impacted family using url (not clean)
- Doing mock updates by chain only

It is a choice but indeed that way it would be cleaner and easier to deal with

Do you think of a better solution ?

Reply:
yes, I see. 
Maybe keep it like this at the moment and think about improvements later.